### PR TITLE
Feature/renovate UI variables

### PR DESCRIPTION
**Pull request by boddissattva**

### CODE REVIEW

**Comment by boddissattva** [comment]
Chutney - Mozilla Firefox 2021-02-17 11-15-09

**Comment by nbrouand** [comment]
When adding a new group, the json should be empty.

File: server/src/main/java/com/chutneytesting/execution/domain/campaign/CampaignExecutionEngine.java [hunk 1]
```diff
@@ -61,8 +61,6 @@
     private final DataSetHistoryRepository dataSetHistoryRepository;
     private final JiraXrayPlugin jiraXrayPlugin;
     private final ChutneyMetrics metrics;
-    private ExecutorService asyncExecutor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors());
```
Comment (iguissouma) [inline server/src/main/java/com/chutneytesting/execution/domain/campaign/CampaignExecutionEngine.java]: I added this to execute code asynchronously for new UI, was it removed?

File: nx/libs/ui-commons/src/lib/components/page-header/page-header.component.scss [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 chutney-page-header-action {
```
Comment (iguissouma) [inline nx/libs/ui-commons/src/lib/components/page-header/page-header.component.scss:16]: how it's shown for other pages?

**Comment by iguissouma** [comment]
I'm not sure about multiple floating action buttons, I think its not compliant with material spec...
see the link:
https://material.io/components/buttons-floating-action-button#behavior

**Comment by iguissouma** [comment]
Add popup to confirm delete

**Pull request closed without merging**